velox/connectors/Connector.h

```diff
@@ -60,6 +60,7 @@ class ColumnHandle {
 class ConnectorTableHandle {
  public:
   virtual ~ConnectorTableHandle() = default;
+  virtual const std::string& name() const = 0;
```

Comment:
Should we add a more generic toString() method instead?

Reply:
good idea!